configurator/src/main.rs

```diff
@@ -809,23 +765,24 @@ fn get_stats(
     peer_tor_address: String,
     alias: String,
 ) -> Result<Properties, anyhow::Error> {
+    println!("calling getinfo...");
     let mut macaroon_file = File::open("/root/.lnd/data/chain/bitcoin/mainnet/admin.macaroon")?;
     let mut macaroon_vec = Vec::with_capacity(macaroon_file.metadata()?.len() as usize);
-    let tls_cert = std::fs::read_to_string("/root/.lnd/tls.cert")?;
+    let tls_cert = std::fs::read_to_string("/mnt/cert/control.cert.pem")?;
     macaroon_file.read_to_end(&mut macaroon_vec)?;
     let mac_encoded = hex::encode_upper(&macaroon_vec);
     let node_info: Option<LndGetInfoRes> = serde_json::from_slice(
         &std::process::Command::new("curl")
             .arg("--no-progress-meter")
-            .arg("--cacert")
-            .arg("/root/.lnd/tls.cert")
             .arg("--header")
             .arg(format!("Grpc-Metadata-macaroon: {}", mac_encoded))
-            .arg("https://lnd.embassy:8080/v1/getinfo")
+            .arg("http://localhost:8081/v1/getinfo")
             .output()?
             .stdout,
     )
     .or::<anyhow::Error>(Ok(None))?;
+
+    println!("writing stats.yaml...");
```

Comment:
why do we need to write this file if properties are now using JS procedures?

Reply:
is stats.yaml used anywhere else in the OS or for the wrapper? if not, I can comment it out